wfe/wfe.go

```diff
@@ -154,14 +165,24 @@ func New(
 	}
 	log.Printf("Configured to reject %d%% of good nonces", nonceErrPercent)
 
+	// Get authz reuse percent from the environment
+	authzReusePercent := defaultAuthzReuse
+	if val, err := strconv.ParseInt(os.Getenv(authzReuseEnvVar), 10, 0); err == nil &&
+		val >= 0 && val <= 100 {
+		authzReusePercent = int(val)
+	}
+	log.Printf("Configured to attempt authz reuse for each identifiers %d%% of the time",
```

Comment:
I think this is a small typo
```suggestion
	log.Printf("Configured to attempt authz reuse for each identifier %d%% of the time",
```